### PR TITLE
Redeem widget 

### PR DESCRIPTION
**Pull request by pannga**

### CODE REVIEW

**Comment by pannga** [comment]
I've prepated new redeem widget with changes from Publish Widget

File: packages/react-widget/src/VoucherifyRedeem.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
+import * as React from 'react'
```
Comment (darekg11) [inline packages/react-widget/src/VoucherifyRedeem.tsx:1]: In old SDK `amount` field is not actually required to be entered even after enabling it.

Comment (darekg11) [inline packages/react-widget/src/VoucherifyRedeem.tsx:1]: Try widget configuration without setting `amount` property and you won't be able to redeem voucher at all.

**Review by darekg11 — CHANGES_REQUESTED** [review]
Changes requested:
- 2 small improvements
- 1 logic error

**Comment by darekg11** [comment]
Oh and if you try to redeem code with white space such as: `TEST THIS` it will convert it to `TESTTHIS` and fail. Is it also happening with publish widget?

**Comment by pannga** [comment]
Updated. As for this voucher error -> I used wrong regex for this, as it was removing spaces from the string. I'm using this regex in Publish Widget but only for phone numbers.

**Review by darekg11 — APPROVED** [review]
Thanks, working great.

**Pull request merged**